topology/University of Wisconsin Milwaukee/UWM - NEMO/NEMO.yaml

```
@@ -29,3 +29,28 @@ Resources:
         Description: UWM-NEMO Hosted CE
         Details:
           hidden: false
+  UWM-NEMO:
+    ID: 1079
+    Disable: false
```

```suggestion
    Active: true
```

Presumably this CE is ready to go?